src/chapter-content/chapter_1_vectors.html

```diff
@@ -4,7 +4,7 @@ <h1>Chapter 1. Vectors</h1>
   <p>“Roger, Roger. What’s our vector, Victor?”</p>
   <p>— Captain Oveur (Airplane)</p>
 </blockquote><a data-type="indexterm" data-primary="vectors"></a>
-<p>This book is all about looking at the world around us and coming up with clever ways to simulate that world with code. Divided into three parts, the book will start by looking at basic physics—how an apple falls from a tree, a pendulum swings in the air, the earth revolves around the sun, etc. Absolutely everything contained within the six chapters of this book requires the use of the most basic building block for programming motion—the <strong><em>vector</em></strong>. And so this is where we begin our story.</p><a data-type="indexterm" data-primary="Euclid"></a><a data-type="indexterm" data-primary="Euclidean vector"></a><a data-type="indexterm" data-primary="geometric vector"></a><a data-type="indexterm" data-primary="vectors" data-secondary="defined"></a>
+<p>This book is all about looking at the world around us and coming up with clever ways to simulate that world with code. Divided into three parts, the book will start by looking at basic physics—how an apple falls from a tree, a pendulum swings in the air, the earth revolves around the sun, etc. Absolutely everything contained within the first five chapters of this book requires the use of the most basic building block for programming motion—the <strong><em>vector</em></strong>. And so this is where I begin our story.</p><a data-type="indexterm" data-primary="Euclid"></a><a data-type="indexterm" data-primary="Euclidean vector"></a><a data-type="indexterm" data-primary="geometric vector"></a><a data-type="indexterm" data-primary="vectors" data-secondary="defined"></a>
```

Comment:
It works! I made some minor edits to Chapter 1 and they came in beautifully when running the GitHub action!

Reply:
Looking great!